AndroidSampleAppWithSdk/README.md

```diff
@@ -4,12 +4,18 @@
 
 Add this in repositories block of app/build.gradle
 ```
-maven { url 'https://locus.bintray.com/locus-lotr-sdk' }
+mavenCentral()
+maven { url 'https://skyhookwireless.github.io/skyhook-location-android' }
```

Comment:
we are not uploading skyhook to maven central, is it?

Reply:
We can't do that because we don't own com.skyhook domain.